controller/src/config.rs

```diff
@@ -19,6 +19,9 @@ pub struct DnsOptions {
     /// however, the email provided here should be a normal "username@domain.tld"-format
     /// email.
     pub soa_email: Option<String>,
+
+    /// The domain name at which the A record for this DNS server is
```

Comment:
I find the wording a bit awkward, how about:

"A domain name whose A record is the IP of this DNS server"?